roles/backend_setup/molecule/default/tests/test_default.py

```diff
@@ -0,0 +1,29 @@
+import os
+
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('all')
+
+
+def test_hosts_file(host):
+    cmd = host.run("vgdisplay")
+    for line in cmd.stdout.split(" "):
+        if "VG Name" in line:
+            assert "vg_vdb" in line
+            assert "vg_vdc" in line
+    cmd = host.run("lvdisplay")
+    for line in cmd.stdout.split(" "):
+        if "LV Path" in line:
+            assert "/dev/vg_vdc/vg_vdc_thinlv1" in line
+            assert "/dev/vg_vdc/vg_vdc_thinlv2" in line
+            assert "/dev/vg_vdb/thicklv_1" in line
+            assert "/dev/centos/swap" in line
+            assert "/dev/centos/root" in line
+        if "LV Name" in line:
+            assert "swap" in line
+            assert "root" in line
```

Comment:
@nigelbabu @Sheersha-jain we can ignore root and swap. No?

Reply:
@sac  Ya can be removed, I removed swap and root from lv names, do you want their paths to be removed from LV Path as well ?

Reply:
Yes, please remove the paths as well, we don't have to verify that.